server/src/controllers/admin.controller.ts

```diff
@@ -15,13 +14,15 @@ export class AdminController {
 
   @Post('load-players')
   async loadPlayers(@Body('players') playersJson: string): Promise<string[]> {
-    const players: Player[] = JSON.parse(playersJson);
+    const players: PlayerData[] = JSON.parse(playersJson);
 
     const successful: string[] = [];
     await Promise.all(
       players.map(async (player) => {
-        if ((await this.playersService.newPlayer(player)) == ResponseStatus.OK)
-          successful.push(player.id);
+        const result = await this.playersService.newPlayer(player);
+        if (result !== undefined) {
```

Comment:
Może warto jednak wrzucać do successful jakiegoś nulla czy coś, jeśli result === undefined? Bo jeśli tak nie zrobimy, i np. klient poprosi o zapisanie 5 graczy, a przy trzecim wystąpi błąd i zostaną zwrócone 4 id, to klient nie będzie wiedział które id jest dla którego gracza